### PR TITLE
Some edits

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**

### CODE REVIEW

File: src/interactive.rs [hunk 1]
```diff
@@ -5,21 +5,25 @@ use colored::Colorize;
...
-    println!("  - govs");
-    println!("  - power");
-    println!("  - usage");
-    println!("  - turbo");
-    println!("  - available_governors");
-    println!("  ");
-    println!("- set");
-    println!("  - gov");
-    println!("  ");
-    println!("E.g. 'get cpus'");
```
Comment (Shuzhengz) [inline src/interactive.rs:22]: lol nice deleting this

**Review by Shuzhengz — APPROVED** [review]
lgtm

File: src/interactive.rs (hunk 1 above)
Comment (JakeRoggenbuck) [reply src/interactive.rs:22]: haha yea, I wrote this like a week ago and i needed to get rid of it

**Review by Camerooooon — APPROVED** [review]
All the changes look nice. I haven't tested yet though.

**Comment by JakeRoggenbuck** [comment]
I tested this a lot

**Pull request merged**